var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -54,7 +54,7 @@ class PyPyside2(PythonPackage):
     depends_on("cmake@3.1:", type="build")
     # libclang versioning from sources/shiboken2/doc/gettingstarted.rst
     depends_on("llvm@6", type="build", when="@5.12:5.13")
-    depends_on("llvm@10", type="build", when="@5.15")
+    depends_on("llvm@10:", type="build", when="@5.15")
```

Comment:
Is LLVM not needed to build `develop`?

Reply:
I changed that line to llvm 10 or greater so I could concretize a spec that needed llvm 11. I have tried building py-pyside2 with a few newer llvm versions and they build ok

Reply:
I mean to build `py-pyside@develop`. Shouldn't this be:
```suggestion
    depends_on("llvm@10:", type="build", when="@5.15:")
```